README.md

```diff
@@ -406,6 +406,8 @@ Getting metadata on any connection returns the following data structure:
 
 The following example illustrates how to use the `getMetadata` method.
 
+Please see the documentation for `Client.getMetadata` for a potential side-effect of the call.
```

Comment:
ok, made it a bit more explicit, less scary ;-) but still pointing to the docs. Duplication is evil ...